modules/social_features/social_user/social_user.tokens.inc

```diff
@@ -37,20 +36,13 @@ function social_user_tokens($type, $tokens, array $data, array $options, Bubblea
   $replacements = [];
 
   if ($type == 'social_user' && !empty($data['message'])) {
```

Comment:
Does it make sense to check for `$data['message']` while it's not being used here?